README.md

```diff
@@ -1 +1,46 @@
-# tech-academy
\ No newline at end of file
+
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Project Name
+
+> Tech Academy
+
+An online website for a Tech academy that show the available courses and the top instructors   
+
+
+## Built With
+
+- HTML ,CSS, JavaScript
+
+
+## Getting Started
+
+To get a local copy up and running follow these simple steps.
+
+git clone git@github.com:Zainab-Alzubair/tech-academy.git
+
+## Online version
+https://zainab-alzubair.github.io/tech-academy/
+## Authors
+
+👤 **Author1**
+
+- GitHub: [@githubhandle](https://github.com/Zainab-Alzubair)
+- Twitter: [@twitterhandle](@zainabm34401029)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/zainab-al-zubair-bb6777168/)
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
```

Comment:
- [ ] Kindly correct your link to your issues page as shown below :+1:

`Feel free to check the [issues page](https://github.com/Zainab-Alzubair/tech-academy/issues).`